src/lib/masq.js

```diff
@@ -98,7 +98,7 @@ const dispatchMasqError = (errorCode) => {
 }
 
 class Masq {
-  constructor () {
+  constructor (dbNameSuffix = '') {
```

Comment:
Workaround in order to avoid conflicts between dbNames during tests, because each HyperDB instance must have its own storage

Reply:
not anymore at least, thanks